command_mode.c

```diff
@@ -2586,7 +2588,7 @@ struct command commands[] = {
 	{ "pl-export",             cmd_pl_export,        1, -1, NULL,                 0, 0          },
 	{ "pl-import",             cmd_pl_import,        0, -1, NULL,                 0, 0          },
 	{ "pl-rename",             cmd_pl_rename,        1, -1, NULL,                 0, 0          },
-	{ "push",                  cmd_push,             1, -1, expand_commands,      0, 0          },
+	{ "push",                  cmd_push,             0, -1, expand_commands,      0, 0          },
```

Comment:
You'll also want to update the [documentation](https://github.com/cmus/cmus/blob/b2921a24357299b3771da5eecfd68465e1280e97/Doc/cmus.txt#L515).

Reply:
Thanks will do.